### PR TITLE
use capacitor AppLauncher instead of Browser

### LINKED ISSUE #686: `Open in Nostr` / `Open in Primal` buttons don't work on iOS app

### PR DESCRIPTION
**Pull request by futurepaul**
closes #686, thx for pointing out this capacitor library @TonyGiorgio 

also remove the hard-coded primal option... on android it prompts anyway

tested on ios and android and web

### CODE REVIEW

**Comment by benthecarman** [comment]
CI failing

**Comment by TonyGiorgio** [comment]
Does it still work with the URL redirect?

File: src/utils/openLinkProgrammatically.ts [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+export async function openLinkProgrammatically(url?: string) {
+    if (!url) return;
     if (Capacitor.isNativePlatform()) {
-        Browser.open({ url: href || "", windowName: "_blank" });
+        const { value } = await AppLauncher.canOpenUrl({ url });
+
+        if (!value) {
+            // Try to open in browser just in case that works glhf
+            window.open(url || "", "_blank");
+            return;
```
Comment (TonyGiorgio) [inline src/utils/openLinkProgrammatically.ts:12]: Actually should probably throw an error or something instead? On iOS I get nothing visual displayed when I don't have a nostr client installed. Just these error logs: 

```
⚡️  To Native ->  AppLauncher canOpenUrl 37902078
⚡️  TO JS {"value":false}
Failed to open URL nostr+walletconnect://d8f8dd6eede3b8b6ed758797f0f463925382c004711f58ae07ff8f6da0b4629f?relay=wss%3A%2F%2Fnostr.mutinywallet.com%2F&secret=7d4041df706206caa145882ac56b3da550d84d67fd5d25358fa5f3a791590c9b: Error Domain=NSOSStatusErrorDomain Code=-10814 "(null)" UserInfo={_LSLine=268, _LSFunction=-[_LSDOpenClient openURL:fileHandle:options:completionHandler:]}
```

Comment (futurepaul) [reply src/utils/openLinkProgrammatically.ts:25]: okay this should be good now

File: src/i18n/en/translations.ts [hunk 2]
```diff
@@ -326,6 +326,9 @@ export default {
             open_app: "Open App",
             open_in_nostr_client: "Open in Nostr Client",
             open_in_primal: "Open in Primal",
+            nostr_client_not_found: "Nostr client not found",
+            client_not_found_description:
+                "Install a nostr client like Primal, Amethyst, or Damus to open this link.",
```
Comment (benthecarman) [inline src/i18n/en/translations.ts:331]: nit: we could make an android and ios specific string

**Review by TonyGiorgio — APPROVED** [review]

**Pull request merged**